distribution/environments/aws-ec2-cloud/config/as-code/artifact-manager-s3.yaml

```diff
@@ -1,14 +1,14 @@
 ---
-# TODO: file JIRAs to make this classifed (and for other used plugins too)
 unclassified:
-  s3blobstoreconfig:
-    # TODO: create through CloudFormation
-    container: "${ARTIFACT_MANAGER_S3_BUCKET_NAME}"
+  artifactManager:
+    artifactManagerFactories:
+      - jclouds:
+          provider: s3
+aws:
+  awsCredentials:
     # FIXME: not a sensible default for everyone
     region: "us-east-1"
+  s3:
+    # TODO: create through CloudFormation
+    container: "${ARTIFACT_MANAGER_S3_BUCKET_NAME}"
     prefix: "jenkins_data/"
```

Comment:
BTW I would suggest the prefix be empty; not sure why we would want to specify one. But up to @carlossg to comment.

Reply:
I think it is useful to allow putting other data in the same bucket. I would have called it `artifacts/` though

Reply:
There is already a subdirectory for artifacts, as well as one for stashes. Currently these are under the build prefix. Maybe we would want to call the default prefix `job/` or `builds/` or something like that then? Or did we want a _top-level` subdirectory for artifacts and one for stashes, to make it easier to apply S3 retention rules—is it possible to do these with glob-type matches or is that unreliable?